package.json

```diff
@@ -11,6 +11,9 @@
 		"type": "git",
 		"url": "https://github.com/dojo/loader.git"
 	},
+	"dependencies": {
+		"dojo-core": "dojo/core"
```

Comment:
But then we either end up relying on another package for implementations to test the loader plugin API with in this package, which seems awkward, or we need to write other loader plugin API tests (presuming that we were relying on the text plugin to test some parts of it in this PR).